sv_inline.h

```diff
@@ -1003,6 +1003,89 @@ Perl_sv_setpv_freshbuf(pTHX_ SV *const sv)
     return SvPVX(sv);
 }
 
+/* int2str_table: lookup table containing string representations of all
+ * two digit numbers. For example, int2str_table.arr[0] is "00" and
+ * int2str_table.arr[12*2] is "12".
+ *
+ * We are going to read two bytes at a time, so we have to ensure that
+ * the array is aligned to a 2 byte boundary. That's why it was made a
+ * union with a dummy U16 member. */
+static const union {
+    char arr[200];
+    U16 dummy;
+} int2str_table = {{
```

Comment:
Will this symbol be visible in external (C/XS) code? Because at the moment it is an unprefixed identifier (no `Perl_` or anything) in the global namespace.

Reply:
Good question. I'm not sure how best to handle it. Happy to take advice.

Reply:
Guarding it inside a `#if defined(PERL_CORE) || defined (PERL_EXT)`?

Reply:
Why not just rename it?

Reply:
Ok, I'll give it a `Perl_` prefix.